stackstate_checks_base/stackstate_checks/base/checks/base.py

```diff
@@ -1106,6 +1112,19 @@ def _submit_metric(self, mtype, name, value, tags=None, hostname=None, device_na
 
         aggregator.submit_metric(self, self.check_id, mtype, ensure_unicode(name), value, tags, hostname)
 
+    def _submit_raw_metrics_data(self, name, value, tags=None, hostname=None, device_name=None, timestamp=None):
+        tags = self._normalize_tags_type(tags, device_name, name)
+
+        if timestamp is None:
+            self.warning('Raw Metric: Timestamp is invalid')
```

Comment:
`Raw Metric: Timestamp is invalid`

`Timestamp is empty, not sending metric: x` something like that. Just to make it clear it's not being sent and it's empty rather than invalid